experiments/functional/blockdevice-reusability/run_litmus_test.yml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: verify-blockdevice-reusability-
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: verify-blockdevice-reusability
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: IfNotPresent
+
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays, actionable, default
+            value: default
+
+          - name: SPC_POOL_NAME
+            value: cstor-spc-disk-pool
+
+            # Provide the cStor CSPC pool name
+          - name: CSPC_POOL_NAME
+            value: cstor-cspc-disk-pool
+
+           # Provide the value for POOL_TYPE 
+           # stripe, mirror, raidz, raidz2
+          - name: POOL_TYPE
+            value: stripe
+
+           # Namespace where the OpenEBS components are deployed
+          - name: OPERATOR_NS
+            value: openebs
+
+        command: ["/bin/bash"]
+        args: ["-c", "ansible-playbook ./experiments/functional/blockdevice-reusability/test.yml -i /etc/ansible/hosts -v; exit 0"]
```

Comment:
can we move this test inside cspc-pool directory rather than keeping it in generic functional directory